### DISCUSSION TITLE
After changing the height and width CSS of the node dynamically, edge position not updating.

### DISCUSSION THREAD

**Discussion by patil-naresh** [xyflow/xyflow#2319] [Q&A]
Updating the height and width CSS properties of the node not updating the edge position.

Initially on load node: -
![Screenshot from 2022-07-26 18-24-07](https://user-images.githubusercontent.com/47170440/181010915-81b6c1ff-2d9f-49b8-984e-8f79f7264cf5.png)

After changing the node width and height:- 

![Screenshot from 2022-07-26 18-24-21](https://user-images.githubusercontent.com/47170440/181011074-4794254f-496e-497e-9f82-ebd54a62820d.png)

**Comment by moklick** [comment]
This looks like a bug. React Flow observes these kind of changes and should update the handles accordingly. Could you create a codesandbox or repo for your example?
